mne/viz/ui_events.py

```diff
@@ -78,17 +76,114 @@ class TimeChange(UIEvent):
 
     Attributes
     ----------
-    name : str
-        The name of the event: ``"time_change"``
-    source : matplotlib.figure.Figure | Figure3D
-        The figure that published the event.
+    %(ui_event_name_source)s
     time : float
         The new time in seconds.
     """
 
     time: float
 
 
+@dataclass
+@fill_doc
+class PlaybackSpeed(UIEvent):
+    """Indicates that the user has selected a different playback speed for videos.
+
+    Parameters
+    ----------
+    speed : float
+        The new speed in seconds per frame.
+
+    Attributes
+    ----------
+    %(ui_event_name_source)s
+    speed : float
+        The new speed in seconds per frame.
+    """
+
+    speed: float
+
+
+@dataclass
+@fill_doc
+class ColormapRange(UIEvent):
+    """Indicates that the user has updated the bounds of the colormap.
+
+    Parameters
+    ----------
+    %(fmin_fmid_fmax)s
+    %(alpha)s
+
+    Attributes
+    ----------
+    %(ui_event_name_source)s
+    %(fmin_fmid_fmax)s
+    %(alpha)s
+    """
+
+    fmin: float = None
+    fmax: float = None
+    fmid: float = None
+    alpha: bool = None
```

Comment:
Since you set None as default, the type annotation should reflect None as a valid value too, either via a Union or Optional